hydroengine_service/main.py

```diff
@@ -800,14 +802,16 @@ def get_water_network():
 
     j = request.json
 
+    use_url = j['use_url']
```

Comment:
Maybe make it more explicit, for example: 

"output_type": ['geojson', 'geojson_url']

Reply:
My implementation is already applied in other places of the code, and thus makes things more uniform. I agree that your approach is more nice, but it lacks consistency.

Reply:
Consistency with what? This approach is more consistent since you won't need to add another parameter if we add export tot GDrive, storage buckets, etc.

Reply:
Consistency with the current code: https://github.com/openearth/hydro-engine-service/blob/master/hydroengine_service/main.py#L580

Reply:
Ah, ok, it was already like that :)